src/SapiStatsDispatcherInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Drupal\sapi;
+
+use \Drupal\sapi\StatisticsItemInterface;
+
+/**
+ * Interface SapiStatsDispatcherInterface.
+ *
+ * @package Drupal\sapi
+ */
+interface SapiStatsDispatcherInterface {
+
+   /**
+    * Creates plugin instances.
```

Comment:
The documentation should say only the main point of the action. My best guess is that `dispatch()` method is best described as `Dispatches the statistics item to interested parties.`